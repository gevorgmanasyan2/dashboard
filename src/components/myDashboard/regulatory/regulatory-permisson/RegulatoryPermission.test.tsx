/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import RegulatoryPermissionList from './RegulatoryPermissions'

it('should render RegulatoryPermissionList component', () => {
  expect(shallow(<RegulatoryPermissionList />)).toMatchSnapshot()
})
