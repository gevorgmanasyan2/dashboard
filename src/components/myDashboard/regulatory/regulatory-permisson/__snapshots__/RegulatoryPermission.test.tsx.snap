// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RegulatoryPermissionList component 1`] = `
<div
  className="regList__container"
>
  <header
    className="regList__container__header"
  >
    <Title
      titleText="Regulatory Permissions"
    />
    <div
      className="regList__container__header--icon"
    >
      <ForwardRef(PlusOutlined) />
    </div>
  </header>
  <Space
    style={
      Object {
        "marginBottom": 16,
      }
    }
  >
    <ShareableSelect
      value="All Sources"
    />
    <ShareableSelect
      value="All Assigned Agents"
    />
    <ShareableSelect
      value="All Waiver Types"
    />
    <ShareableSelect
      value="Active"
    />
  </Space>
  <Connect(Regulatory) />
</div>
`;
